### PR TITLE
New elk

### PR DESCRIPTION
**Pull request by sebasampaoli**

### CODE REVIEW

File: modules/ROOT/nav.adoc [hunk 1]
```diff
@@ -31,4 +31,5 @@
  ** xref:access-management-disclaimer.adoc[Configure a Disclaimer for Anypoint Platform PCE]
  ** xref:custom-policies.adoc[Upload Custom Policies and Publish Assets to Exchange Using the Maven Client]
  ** xref:restarting-a-node.adoc[Update Nodes]
+ ** xref:ext-analytics-elk.adoc[Analyze Business and API Data using ELK]
```
Comment (kbostrom) [inline modules/ROOT/nav.adoc]:
```suggestion
 ** xref:ext-analytics-elk.adoc[Analyze Business and API Data Using ELK]
```

File: modules/ROOT/pages/ext-analytics-elk.adoc [hunk 2]
```diff
@@ -0,0 +1,173 @@
+= Analyze Business and API Data using ELK
```
Comment (kbostrom) [inline modules/ROOT/pages/ext-analytics-elk.adoc]:
```suggestion
= Analyze Business and API Data Using ELK
```

File: modules/ROOT/pages/ext-analytics-elk.adoc [hunk 3]
```diff
@@ -0,0 +1,173 @@
...
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Use the Elastic Stack (ELK) to analyze the business data and API analytics generated by Anypoint Platform Private Cloud Edition (Anypoint Platform PCE). You can use Filebeat to process Anypoint Platform log files, insert them into an Elasticsearch database, and then analyze them with Kibana.
```
Comment (kbostrom) [inline modules/ROOT/pages/ext-analytics-elk.adoc]: I found this when I looked up ELK for our Term Presentation List:

The **ELK stack** is an acronym used to describe a stack that comprises of three popular open-source projects: Elasticsearch, Logstash, and Kibana.

Do you support my listing of this term as "ELK stack"?

Comment (kbostrom) [reply modules/ROOT/pages/ext-analytics-elk.adoc]: Is it okay that this narrative does not mention Logstash?

File: modules/ROOT/pages/ext-analytics-elk.adoc [hunk 4]
```diff
@@ -0,0 +1,173 @@
...
+
+The following image shows an example of how API data appears in Kibana:
```
Comment (kbostrom) [inline modules/ROOT/pages/ext-analytics-elk.adoc]:
```suggestion
The following example shows how API data appears in Kibana:
```

File: modules/ROOT/pages/ext-analytics-elk.adoc [hunk 5]
```diff
@@ -0,0 +1,173 @@
...
+
+image::kibana-example.png[]
+
+== Prerequisites
+
+* Required software
```
Comment (kbostrom) [inline modules/ROOT/pages/ext-analytics-elk.adoc:14]: Suggest making this Level-1 list item parallel with its peers, either by making all three fragments or all three sentences.

File: modules/ROOT/pages/ext-analytics-elk.adoc [hunk 6]
```diff
@@ -0,0 +1,173 @@
...
++
+To export external analytics data to ELK, you must have the following software:
++
+** Mule Runtime, version 3.8.4 or later
```
Comment (kbostrom) [inline modules/ROOT/pages/ext-analytics-elk.adoc]: GLOBAL, proper name is Mule runtime engine (Mule):
```suggestion
** Mule runtime engine (Mule) version 3.8.4 or later
```

File: modules/ROOT/pages/ext-analytics-elk.adoc [hunk 7]
```diff
@@ -0,0 +1,173 @@
...
+** Mule Runtime Manager Agent, version 1.7.0 or later
```
Comment (kbostrom) [inline modules/ROOT/pages/ext-analytics-elk.adoc]: Also GLOBAL:
```suggestion
** Anypoint Runtime Manager agent version 1.7.0 or later
```

Comment (kbostrom) [reply modules/ROOT/pages/ext-analytics-elk.adoc]: Globally remove commas before version info

File: modules/ROOT/pages/ext-analytics-elk.adoc [hunk 8]
```diff
@@ -0,0 +1,173 @@
...
+** Elasticsearch, version 7.6.2
+** Filebeat, version 7.6.2
+** Kibana, version 7.6.2
+
+* Ensure that you have installed and configured your Mule runtime engine instances.
+* Ensure that you have registered these instances with Runtime Manager.
```
Comment (kbostrom) [inline modules/ROOT/pages/ext-analytics-elk.adoc]: See comment above re. peer list items being parallel.

File: modules/ROOT/pages/ext-analytics-elk.adoc (hunk 3 above)
Comment (kbostrom) [reply modules/ROOT/pages/ext-analytics-elk.adoc]: Do we have anything to say about WHY your audience wants or needs these products and processes, or why they in particular are superior?

File: modules/ROOT/pages/ext-analytics-elk.adoc [hunk 9]
```diff
@@ -0,0 +1,173 @@
...
+
+== Edit the wrapper.conf Properties File
+
```
Comment (kbostrom) [inline modules/ROOT/pages/ext-analytics-elk.adoc:28]: Why?

File: modules/ROOT/pages/ext-analytics-elk.adoc [hunk 10]
```diff
@@ -0,0 +1,173 @@
...
+In your Mule runtime installation, edit the following properties in the `conf/wrapper.conf` file:
+
+. Set the `analytics_enabled` property to `true`.
```
Comment (kbostrom) [inline modules/ROOT/pages/ext-analytics-elk.adoc]: GLOBALLY end text that introduces an example, list, table, and so on with a colon:
```suggestion
. Set the `analytics_enabled` property to `true`:
```

Comment (kbostrom) [reply modules/ROOT/pages/ext-analytics-elk.adoc]: Please repeat this punctuation for each step that follows.

File: modules/ROOT/pages/ext-analytics-elk.adoc [hunk 11]
```diff
@@ -0,0 +1,173 @@
...
+----
+
+== Enable Event Tracking
+
+To export business and event tracking data, you must enable event tracking in Runtime Manager:
+
+. From Anypoint Platform, select *Runtime Manager*.
+. Select *Servers*.
+. Select the row containing the server on which you want to configure ELK.
+. Select *Manager Server* and then select the Plugins tab.
```
Comment (kbostrom) [inline modules/ROOT/pages/ext-analytics-elk.adoc]:
```suggestion
. Select *Manager Server* and then select the *Plugins* tab.
```

File: modules/ROOT/pages/ext-analytics-elk.adoc [hunk 12]
```diff
@@ -0,0 +1,173 @@
...
+. In Event Tracking, select *Business Events* from the *Levels* drop-down menu.
```
Comment (kbostrom) [inline modules/ROOT/pages/ext-analytics-elk.adoc]:
```suggestion
. In *Event Tracking*, select *Business Events* from the *Levels* drop-down menu.
```

File: modules/ROOT/pages/ext-analytics-elk.adoc [hunk 13]
```diff
@@ -0,0 +1,173 @@
...
+. Select the switch to enable event tracking on ELK.
++
+Runtime Manager displays the ELK Integration dialog.
```
Comment (kbostrom) [inline modules/ROOT/pages/ext-analytics-elk.adoc]:
```suggestion
Runtime Manager displays the *ELK Integration* dialog.
```

File: modules/ROOT/pages/ext-analytics-elk.adoc [hunk 14]
```diff
@@ -0,0 +1,173 @@
...
+. Select *Apply*.
+
+== Enable API Analytics
+
+To export API analytics, you must enable API analytics in Runtime Manager.
+
+. From Anypoint Platform, select *Runtime Manager*.
+. Select *Servers*.
+. Select the row containing the server on which you want to configure ELK.
+. Select the switch to enable API analytics and then select the switch next to ELK.
```
Comment (kbostrom) [inline modules/ROOT/pages/ext-analytics-elk.adoc]:
```suggestion
. Select the switch to enable API analytics and then select the switch next to *ELK*.
```

File: modules/ROOT/pages/ext-analytics-elk.adoc [hunk 15]
```diff
@@ -0,0 +1,173 @@
...
+. Configure the log files as necessary.
+. Select *Apply*.
+
+== Install and Configure the Filebeat Agent
+
+On each server with Mule runtime instances, download and install Filebeat for your operating system.
```
Comment (kbostrom) [inline modules/ROOT/pages/ext-analytics-elk.adoc:78]:
```suggestion
On each server with Mule instances, download and install Filebeat for your operating system.
```

File: modules/ROOT/pages/ext-analytics-elk.adoc [hunk 16]
```diff
@@ -0,0 +1,173 @@
...
+
+== Configure the Mule Filebeat Module
+
+On each server with Mule runtime instances, download and expand the Filebeat Module archive from the following URL:
```
Comment (kbostrom) [inline modules/ROOT/pages/ext-analytics-elk.adoc:82]:
```suggestion
On each server with Mule instances, download and expand the Filebeat Module archive from the following URL:
```

File: modules/ROOT/pages/ext-analytics-elk.adoc [hunk 17]
```diff
@@ -0,0 +1,173 @@
...
++
+----
+https://s3.us-east-2.amazonaws.com/elk-integration/elk-integration-06-08-20/elk-integration.tar.gz
+----
+
+. Add the absolute paths of each log file to the `manifest.yml` files:
++
+* Inside the `businessevents` folder, set the event log paths to the `var` variable of your operating system (OS), as shown in the following example:
```
Comment (kbostrom) [inline modules/ROOT/pages/ext-analytics-elk.adoc]: Colon renders "as shown..." text unnecessary; GLOBAL:
```suggestion
* Inside the `businessevents` folder, set the event log paths to the `var` variable of your operating system (OS):
```

File: modules/ROOT/pages/ext-analytics-elk.adoc [hunk 18]
```diff
@@ -0,0 +1,173 @@
...
+  - name: paths
+    default:
+      - /var/mule/logs/events.log
+    os.darwin:
+      - /var/mule/logs/events.log
+    os.windows:
+      - /var/mule/logs/events.log
+----
++
+* Inside the `apianalytics` folder, set the API analytics log paths to the `var` variable of your OS, as shown in the following example:
```
Comment (kbostrom) [inline modules/ROOT/pages/ext-analytics-elk.adoc]: ditto:
```suggestion
* Inside the `apianalytics` folder, set the API analytics log paths to the `var` variable of your OS:
```

File: modules/ROOT/pages/ext-analytics-elk.adoc [hunk 19]
```diff
@@ -0,0 +1,173 @@
...
+  -E setup.template.overwrite=true
+----
+
+== Configure Kibana to Load an Index
+
+After installing Filebeat and Elasticsearch, you must configure Kibana to consume data from Anypoint Platform.
+
+MuleSoft provides a default Kibana configuration that you can use to analyze business and API data. This configuration includes dashboards, searches, and visualizations.
+
+. Configure an Index Pattern
```
Comment (kbostrom) [inline modules/ROOT/pages/ext-analytics-elk.adoc]: Lowercase per search:
```suggestion
. Configure an index pattern.
```

File: modules/ROOT/pages/ext-analytics-elk.adoc [hunk 20]
```diff
@@ -0,0 +1,173 @@
...
++
+You must create an Elasticsearch index for Anypoint Platform data.
```
Comment (kbostrom) [inline modules/ROOT/pages/ext-analytics-elk.adoc]:
```suggestion
You must create an Elasticsearch index for Anypoint Platform data:
```

**Review by kbostrom — APPROVED** [review]
Edited

File: modules/ROOT/pages/ext-analytics-elk.adoc [hunk 21]
```diff
@@ -0,0 +1,174 @@
+= Analyze Business and API Data Using ELK
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Use the Elastic Stack (ELK stack) to analyze the business data and API analytics generated by Anypoint Platform Private Cloud Edition (Anypoint Platform PCE). You can use Filebeat to process Anypoint Platform log files, insert them into an Elasticsearch database, and then analyze them with Kibana.
+
+The following example shows how API data appears in Kibana:
+
+image::kibana-example.png[]
```
Comment (federicoboerr) [inline modules/ROOT/pages/ext-analytics-elk.adoc:10]: Is this image working? I can't see it in https://github.com/mulesoft/docs-private-cloud/blob/new-elk/modules/ROOT/pages/ext-analytics-elk.adoc

Comment (federicoboerr) [reply modules/ROOT/pages/ext-analytics-elk.adoc:10]: @sebasampaoli do we need to update it?

File: modules/ROOT/pages/ext-analytics-elk.adoc [hunk 22]
```diff
@@ -0,0 +1,174 @@
...
+. Select *Apply*.
+
+== Install and Configure the Filebeat Agent
+
+On each server with Mule runtime instances, download and install Filebeat for your operating system.
+
+== Configure the Mule Filebeat Module
+
+On each server with Mule runtime instances, download and expand the Filebeat Module archive from the following URL:
++
```
Comment (federicoboerr) [inline modules/ROOT/pages/ext-analytics-elk.adoc]: Does this render ok in docs?
image

**Review by federicoboerr — APPROVED** [review]
Minor comments on format, rest looks good.

File: modules/ROOT/pages/ext-analytics-elk.adoc (hunk 21 above)
Comment (sebasampaoli) [reply modules/ROOT/pages/ext-analytics-elk.adoc:10]: No need to update it, the image works well as an example.

**Pull request merged**